### PR TITLE
[TIMOB-20030] iOS: Removing repeated eventListeners

### PR DESCRIPTION
**Pull request by AngelkPetkov**
:JIRA: https://jira.appcelerator.org/browse/TIMOB-20030?filter=-1

### CODE REVIEW

File: iphone/Classes/KrollObject.m [hunk 1]
```diff
@@ -1347,7 +1347,17 @@ -(void)storeListener:(id)eventCallbackOrWrapper forEvent:(NSString *)eventName
 	{
 		TiValueRef jsCallbackArrayLength = TiObjectGetProperty(jsContext, jsCallbackArray, kTiStringLength, &exception);
 		int arrayLength = (int)TiValueToNumber(jsContext, jsCallbackArrayLength, &exception);
-		
+        
+		double lenght = TiValueToNumber(jsContext, jsCallbackArrayLength, NULL);
```
Comment (cheekiatng) [inline iphone/Classes/KrollObject.m]: isn't this the same as arrayLength that was previously declared? please explain why this has to be declared again?
and length has to be spelt correctly.

**Comment by cheekiatng** [comment]
FT passed. Please address comments.

File: iphone/Classes/KrollObject.m (hunk 1 above)
Comment (pec1985) [inline iphone/Classes/KrollObject.m]: Yep, it's the same.... :D

**Comment by cheekiatng** [comment]
CR and FT Passed. approved!

**Pull request merged**